src/server/pps/server/api_server.go

```diff
@@ -1539,3 +1550,7 @@ func (s podSlice) Swap(i, j int) {
 func (s podSlice) Less(i, j int) bool {
 	return s[i].ObjectMeta.Name < s[j].ObjectMeta.Name
 }
+func zeroDuration() time.Duration {
+	d, _ := time.ParseDuration("0s")
+	return d
+}
```

Comment:
time.Duration is just an int64 so I think you can just do `time.Duration(0)` to get the same effect. You might as well inline it rather than defining a function since `zeroDuration` is only a little bit shorter.